### PR TITLE
Feature/Add Backbone compatibility check

### PR DESCRIPTION
**Pull request by Milena-Czierlinski**

### CODE REVIEW

File: packages/enmeshed_types/lib/src/responses/backbone_compatibility.dart [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'backbone_compatibility.g.dart';
+
+@JsonSerializable(includeIfNull: false)
+class CheckBackboneCompatibilityResponse {
+  final bool isCompatible;
+  final dynamic backboneVersion;
+  final dynamic supportedMinBackboneVersion;
+  final dynamic supportedMaxBackboneVersion;
```
Comment (jkoenig134) [inline packages/enmeshed_types/lib/src/responses/backbone_compatibility.dart:10]: these are ints

Comment (jkoenig134) [reply packages/enmeshed_types/lib/src/responses/backbone_compatibility.dart:10]: please never use dynamic without a GOOD reason.

Comment (Milena-Czierlinski) [reply packages/enmeshed_types/lib/src/responses/backbone_compatibility.dart:10]: I had them initially as ints, but that gave me an error saying that type String is not assignable to type num (in the generated fromJson of the CheckBackboneCompatibilityResponse).

Comment (jkoenig134) [reply packages/enmeshed_types/lib/src/responses/backbone_compatibility.dart:10]: what the hell? :O

Comment (jkoenig134) [reply packages/enmeshed_types/lib/src/responses/backbone_compatibility.dart:10]: have you used @IntegerConverter()?

Comment (jkoenig134) [reply packages/enmeshed_types/lib/src/responses/backbone_compatibility.dart:10]: just search it globally, you find the usage.

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**